mods/butterflies/README.txt

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Inconsistent empty line

Reply:
Heh, sorry. I often leave the first line in my own mods. Will adjust.